src/tools/openpgp-tool.c

```diff
@@ -479,14 +588,16 @@ int main(int argc, char **argv)
 	r = util_connect_card(ctx, &card, opt_reader, opt_wait, verbose);
 	if (r) {
 		util_fatal("failed to connect to card: %s\n",
-			sc_strerror(r));
+		           sc_strerror(r));
```

Comment:
Hi @frankmorgner 

How should it be? I thought the second argument should be vertically align with the first argument, like this:
![selection_011](https://cloud.githubusercontent.com/assets/314607/7331210/05905694-eb3a-11e4-8810-ab88cb0b7250.png)